tsconfig.json

```diff
@@ -1,11 +1,18 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es5",
+    "strict": true,
     "module": "esnext",
     "moduleResolution": "node",
-    "strict": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": true,
-    "experimentalDecorators": true
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "types": [
```

Comment:
What's your reasoning for specifying `types` explicitly? I believe this means that @types/* packages are no longer automatically included, which would be a surprise for people who are starting with this boilerplate (see https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).

Reply:
For that, i just followed http://inversify.io/